stl/inc/utility

```diff
@@ -644,6 +644,88 @@ template <size_t _Idx>
 inline constexpr in_place_index_t<_Idx> in_place_index{};
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+template <class _Ty>
+inline constexpr bool _Is_integral_not_bool_char = is_integral_v<_Ty>
+    && !_Is_any_of_v<_Ty, bool, char,

```

Comment:
Semantics: `is_integral_v`, being a primary type category, ignores top-level cv-qualifiers. This `_Is_any_of_v` check, being an extended `is_same` trait, is sensitive to top-level cv-qualifiers. Although it may not matter given your specific usage below, in the past we've found that internal type traits not following Standard type trait conventions can lead to headaches. I recommend using `remove_cv_t<_Ty>` here. (The throughput cost is very low, as the compiler optimizes that trait.)